### ISSUE TITLE
Right clicking question

### ISSUE THREAD

**Issue by Qronikarz** [fredcw/Cinnamenu#5]
Hello.

I have a question.
On normal Cinnamon menu you can right click programs to have some of their options displayed:
![menu](https://github.com/user-attachments/assets/3ef850e4-cac5-40c7-bf33-118681129f08)

On Cinnamenu however it only displays this:
![Cinnamenu](https://github.com/user-attachments/assets/512b4043-d0ba-4cc0-a2a6-d85f8b611554)

I want to know if I can somehow enable the behavior of original menu or is not possible to implement?

The activator properties from original menu aren't available and they would be useful:
![activator](https://github.com/user-attachments/assets/50994678-a2f7-40ef-a1a3-b827995364da)

**Comment by fredcw** [comment]
Yes, I haven't implemented the additional application actions in the context menu because I'm not sure there's much demand for it to make it worthwhile even though it's been done in the default cinnamon menu applet. I also don't really see how it's useful. Steam is the only app that I know of that seems to make much use of it except chrome which has "Open new Window" but all these things you can easily do from the steam app or chrome browser anyway so I don't see the point.

As, for the "Properties" option, you can press <ctrl>p when the app is highlighted to open the properties (see "tips" page in config). I may add it to the context menu because a few people have requested it now. Thanks.

**Comment by Qronikarz** [comment]
> I also don't really see how it's useful
Just a matter of convenience really. Saving few clicks and in case of some apps launching it directly into the tab you want.

> I may add it to the context menu because a few people have requested it now
It would be nice. Pressing P still requires using a keyboard and otherwise you can fully browse it with just mouse.

**Comment by fredcw** [comment]
Sorry, that was meant to be \<crtl\>p. The \<ctrl\> disappeared because I didn't escape the angle brackets (\\\<ctrl\\\>p)

**Comment by umikali222** [comment]
I'm having this same issue. Please add it to the context menu, because right now I have to have both cinnamenu and the default mint menu as I modify the launcher properties quite often.

**Comment by fredcw** [comment]
@umikali222 Thanks. I've added it to the context menu. Latest version is in this repo only.

**Comment by fredcw** [comment]
@Qronikarz Application actions has been added, thanks.
